Rakefile

```diff
@@ -9,7 +9,7 @@ require 'engine_cart/rake_task'
 
 APP_ROOT = '.'
 require 'jettywrapper'
-JETTY_ZIP_BASENAME = 'master'
+JETTY_ZIP_BASENAME = 'fedora-4/master'
```

Comment:
`v8.1.1` is what we want here, or just get rid of the next line and have: `Jettywrapper.hydra_jetty_version = "v8.1.0"
`